benchmarks/ner_embeddings/project.yml

```diff
@@ -0,0 +1,272 @@
+title: "Comparing embedding layers in spaCy"
+description: |
+  This project compares `MultiHashEmbed` with its standard embedding counterpart
+  `MultiEmbed` for Named Entity Recognition. In order to access the datasets,
+  you first need to clone the [`spancat-datasets`](https://github.com/explosion/spancat-datasets)
+  repository:
+
+  ```sh
+  # Clone the spancat-datasets repo to access the datasets
+  git clone git@github.com:explosion/spancat-datasets.git
+  ```
+
+  and run the necessary conversion scripts. For example, let's perform the
+  conversion command for the [Anatomical Entity Mention (AnEM)](http://www.nactem.ac.uk/anatomy/)
+  corpus:
+
```

Comment:
Maybe add a step that you'll need to run `spacy project assets` first, too?

Reply:
Agreed

Reply:
Unagreed, its actually in `projects` now.